### DISCUSSION TITLE
Using multiple optics group with cryodrgn

### DISCUSSION THREAD

**Discussion by Adwaith99** [zhonge/cryodrgn#192] [General]
Hi, 

I have a dataset which has been generated by combining particles from two different datasets using Join star files in Relion. However, when I try to parse poses in cryoDRGN, I get an error saying only one optics group is supported. 

Is there any way around this problem?

Thank you

**Comment by LabZ-muffin** [comment]
Hello, 

I am having the same error, except I am working with a lot more optics groups, Could you solve it ?

Thanks
